vendor/engines/split_accounts/config/locales/en.yml

```diff
@@ -20,3 +20,8 @@ en:
       split_accounts/split_account:
         one: Split Account
         other: Split Accounts
+
+  controllers:
+    general_reports:
+      headers:
+        split_percentage: "Split Percentage"
```

Comment:
Very minor.  If we want to remain consistent with the rest of the `split_accounts` gem lingo, we may want to use `split_percent` instead of `split_percentage`?